### ISSUE TITLE
Feature request: "Layer by xyz..." option

### ISSUE THREAD

**Issue by SebiTimeWaster** [photopea/photopea#2457]
It would be nice to have the options "Create layer by copy" and "Create layer by cut" in the context menu when using the selection tool :)

**Comment by photopea** [comment]
But what about using a good old Ctrl+C (or X) and Ctrl+V? :)

**Comment by SebiTimeWaster** [comment]
yeah, thats fine (and i used it before), but there is nothing wrong with showing the user that he has these options if he does not know that the standard copy and paste also works on masks.
